Compile Error: Use of undeclared crate or module `argonautica`

I followed the quick start to create a new app.

When I run `cargo fullstack` or `cargo build`, I receive the following error:

```
   Compiling create-rust-app v8.1.1
error[E0433]: failed to resolve: use of undeclared crate or module `argonautica`
 --> /Users/rohit/.cargo/registry/src/github.com-1ecc6299db9ec823/create-rust-app-8.1.1/src/dev/endpoints/service_poem.rs:1:5
  |
1 | use argonautica::config::Version::_0x10;
  |     ^^^^^^^^^^^ use of undeclared crate or module `argonautica`
```

Originates here:

https://github.com/Wulf/create-rust-app/blob/50cd54b14d50ac1b733fd653723978518344a1ec/create-rust-app/src/dev/endpoints/service_poem.rs#L1

--- a/create-rust-app/src/dev/endpoints/service_poem.rs
+++ b/create-rust-app/src/dev/endpoints/service_poem.rs
@@ -1,4 +1,3 @@
-use argonautica::config::Version::_0x10;
 use poem::{
     get, handler,
     http::StatusCode,

